packages/tasklist-client-node-js/src/__test__/restclient.spec.ts

```diff
@@ -1,4 +1,4 @@
-import { TaskState } from '../lib/Types'
+import { TaskStateREST } from '../lib/Types'
```

Comment:
See below for the source of this new enum. 

It's been added because the shape of the GraphQL task state is different from the shape of the REST task state. More notes further down.